middle_end/flambda2/terms/flambda_primitive.ml

```diff
@@ -1042,6 +1042,27 @@ let nullary_classify_for_printing p =
   | Begin_try_region | Enter_inlined_apply _ | Dls_get ->
     Neither
 
+module Reinterpret_64_bit_word = struct
+  type t =
+    | Tagged_int63_as_unboxed_int64
+    | Unboxed_int64_as_tagged_int63
+    | Unboxed_int64_as_unboxed_float64
+    | Unboxed_float64_as_unboxed_int64
```

Comment:
Should I lift the float32 conversions to this level as well? Currently float32<->int32-bits is a @@ builtin.

Reply:
I think we probably should, in a follow-up PR.